Clarify that direct_post endpoint response is JSON. (#273)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

The Response Mode is defined in accordance with [@!OAuth.Responses] as follows:

`direct_post`:

> : In this mode, the Authorization Response is sent to the Verifier using an HTTP POST request to an endpoint controlled by the Verifier. The Authorization Response parameters are encoded in the body using the `application/x-www-form-urlencoded` content type. The names and values in the body MUST be encoded using UTF-8. The response from the Verifier is a JSON object as defined below. The flow can end with an HTTP POST request from the Wallet to the Verifier, or it can end with a redirect that follows the HTTP POST request, if the Verifier responds with a redirect URI to the Wallet.

Reviewer: ```suggestion
: In this mode, the Authorization Response is sent to the Verifier using an HTTP POST request to an endpoint controlled by the Verifier. In this mode the Authorization Response MUST be encoded in the request body using the format defined by the `application/x-www-form-urlencoded` HTTP content type. The parameters in the request body MUST all be encoded using UTF-8. The body of the HTTP response to this request MUST be a JSON object as defined below and designated with the HTTP header of `application/json`. The JSON object in the HTTP response body MAY include a `redirect_uri` parameter which is to be executed as a redirect by the wallet to complete the flow.
```

Tried to make this a little clearer, not precious about the changes though.

Author: Refining this a little as I don't want to be normatively defining the response in 2 places:

```suggestion
: In this mode, the Authorization Response is sent to the Verifier using an HTTP POST request to an endpoint controlled by the Verifier. The Authorization Response MUST be encoded in the request body using the format defined by the `application/x-www-form-urlencoded` HTTP content type. The parameters in the request body MUST all be encoded using UTF-8. The verifier can request that the wallet redirects the user to the verifier using the response as defined below.
```

  state=eyJhb...6-sVA
```

> If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTP status code 200 and a JSON object.

Reviewer: ```suggestion
If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with HTTP status code of 200 and a JSON object in the response body.
```

Author: Refined to include the text you'd suggested above:

```suggestion
If the Response URI has successfully processed the Authorization Response or Authorization Error Response, it MUST respond with an HTTP status code of 200 with `Content-Type` of `application/json` and a JSON object in the response body.
```
